packages/routing-utils/src/superstatic.ts

```diff
@@ -3,7 +3,14 @@
  * See https://github.com/firebase/superstatic#configuration
  */
 import { parse as parseUrl, format as formatUrl } from 'url';
-import { Route, Redirect, Rewrite, HasField, Header } from './types';
+import {
+  Route,
+  Redirect,
+  Rewrite,
+  HasField,
+  Header,
+  HasConditionValue,
```

Comment:
Lets avoid export `HasConditionValue` and you can reference with `HasField['value']` I believe